installer/PowerToysBootstrapper/bootstrapper/bootstrapper.cpp

```diff
@@ -230,15 +271,15 @@ int Bootstrapper(HINSTANCE hInstance)
     const VersionHelper myVersion(VERSION_MAJOR, VERSION_MINOR, VERSION_REVISION);
 
     // Do not support installing on Windows < 1903
-    if (updating::is_1809_or_older())
+    //if (updating::is_1809_or_older())
```

Comment:
What is the reason behind commenting it out?

Reply:
thanks, fixed!